README.md

```diff
@@ -89,9 +89,15 @@ A list of test run definitions are provided as [`testconfig.yaml`](testconfig.ya
   numPods: 7
   duration: 5
   hostNetwork: false
-  encap: vxlan
-  dataplane: iptables
-  iterations: 3
+  iterations: 1
+  leaveStandingConfig: true
```

Comment:
i do not think there is explanation for this on the README

Reply:
leaveStandingConfig: https://github.com/projectcalico/tiger-bench/blob/main/README.md?plain=1#L113

iterations: https://github.com/projectcalico/tiger-bench/blob/main/README.md?plain=1#L121